Branch-SDK/src/main/java/io/branch/referral/DeviceInfo.java

```diff
@@ -58,7 +58,7 @@ private DeviceInfo(Context context) {
      *
      * @param requestObj JSON object for Branch server request
      */
-    void updateRequestWithV1Params(ServerRequest serverRequest, JSONObject requestObj) {
+    void updateRequestWithParamsLegacyFormat(ServerRequest serverRequest, JSONObject requestObj) {
```

Comment:
Let's create a separate ticket for this

Reply:
https://branch.atlassian.net/browse/SDK-679